app/helpers/application_helper.rb

```diff
@@ -3,6 +3,16 @@ def service_name
     t('service.name')
   end
 
+  def app_banner_colour
```

Comment:
Any reason to add a helper for banner_colour but not banner_tag or viceversa?

Reply:
The banner tag is only used in one place so doesn't really need a helper method. I also see the banner text as unique for each environment so the method would not really do much but just return the tag whereas the app banner colour is more likely to be duplicated across environments so a method would reduce duplicating the label data - does that make sense?